### PR TITLE
Enable Alpine 3.13 rootfs generation

### PR DESCRIPTION
**Pull request by janvorli**

### CODE REVIEW

**Review by wfurt — APPROVED** [review]
LGTM.
It is good to see that all we need is the main branch now.

File: eng/common/cross/build-rootfs.sh [hunk 1]
```diff
@@ -256,15 +266,19 @@ if [[ "$__CodeName" == "alpine" ]]; then
       -U --allow-untrusted --root $__RootfsDir --arch $__AlpineArch --initdb \
       add $__AlpinePackages
 
-    $__ApkToolsDir/apk-tools-$__ApkToolsVersion/apk \
-      -X http://dl-cdn.alpinelinux.org/alpine/edge/main \
-      -U --allow-untrusted --root $__RootfsDir --arch $__AlpineArch --initdb \
-      add $__AlpinePackagesEdgeMain
+    if [[ ! -z "__AlpinePackagesEdgeMain" ]]; then
```
Comment (am11) [inline eng/common/cross/build-rootfs.sh:269]:
```suggestion
    if [[ -n "$__AlpinePackagesEdgeMain" ]]; then
```

File: eng/common/cross/build-rootfs.sh [hunk 2]
```diff
@@ -256,15 +266,19 @@ if [[ "$__CodeName" == "alpine" ]]; then
...
+        -X http://dl-cdn.alpinelinux.org/alpine/edge/main \
+        -U --allow-untrusted --root $__RootfsDir --arch $__AlpineArch --initdb \
+        add $__AlpinePackagesEdgeMain
+    fi
 
-    $__ApkToolsDir/apk-tools-$__ApkToolsVersion/apk \
-      -X http://dl-cdn.alpinelinux.org/alpine/edge/community \
-      -U --allow-untrusted --root $__RootfsDir --arch $__AlpineArch --initdb \
-      add $__AlpinePackagesEdgeCommunity
+    if [[ ! -z "__AlpinePackagesEdgeCommunity" ]]; then
```
Comment (am11) [inline eng/common/cross/build-rootfs.sh:276]:
```suggestion
    if [[ -n "$__AlpinePackagesEdgeCommunity" ]]; then
```

**Comment by am11** [comment]
>  It is good to see that all we need is the main branch now.

lldb, for instance, is still only available in community repository: https://pkgs.alpinelinux.org/packages?name=lldb&branch=v3.13.

**Comment by janvorli** [comment]
> lldb, for instance, is still only available in community repository

Right, I meant that the edge repo is not needed anymore. Mixing stuff from edge and other version repository was sometimes causing issues.

**Comment by janvorli** [comment]
@am11 thank you for your suggestions, I've applied the changes and fixed the comment (it was saying edge branch instead of edge repo).

File: eng/common/cross/build-rootfs.sh [hunk 3]
```diff
@@ -256,15 +266,19 @@ if [[ "$__CodeName" == "alpine" ]]; then
       -U --allow-untrusted --root $__RootfsDir --arch $__AlpineArch --initdb \
       add $__AlpinePackages
 
-    $__ApkToolsDir/apk-tools-$__ApkToolsVersion/apk \
-      -X http://dl-cdn.alpinelinux.org/alpine/edge/main \
-      -U --allow-untrusted --root $__RootfsDir --arch $__AlpineArch --initdb \
-      add $__AlpinePackagesEdgeMain
+    if [[ -n "__AlpinePackagesEdgeMain" ]]; then
```
Comment (am11) [inline eng/common/cross/build-rootfs.sh:269]: @janvorli, the `$` is missing.
```suggestion
    if [[ -n "$__AlpinePackagesEdgeMain" ]]; then
```

Comment (janvorli) [reply eng/common/cross/build-rootfs.sh]: Oops, thank you for spotting it. Fixed in the last commit.

**Pull request merged**